semopenalex-app/data/templates/http%3A%2F%2Fwww.metaphactory.semopenalex.com%2Fapp%2FStart.html

```diff
@@ -493,72 +498,117 @@
 </template-fragment>
 
 <template-fragment id="overview-works">
-  <div class="row card publications" style='margin:8px'>
   
-<div class="publications__header">
-  <div class="card__title">
-      WORKS
-  </div>
-</div>
-  <semantic-table query=
-    'SELECT ?title ?publication ?cited ?publicationYear ?publisher ?volume ?issue (group_concat(DISTINCT ?name; separator = ", ") as ?authors) WHERE {
-        VALUES ?publication {
-          {{#each bindings}}
-            <{{subject.value}}>
-          {{/each}}
-        }   
-      ?publication <http://purl.org/dc/terms/creator> ?author . 
-      ?publication <http://purl.org/dc/terms/title> ?title .
-      ?author <http://xmlns.com/foaf/0.1/name> ?name .
-      ?publication <https://semopenalex.org/property/citedByCount> ?cited .
-      OPTIONAL {
-        ?publication <https://semopenalex.org/property/hasHostVenue> ?hostVenue .
-        ?hostVenue <https://semopenalex.org/property/hasVenue> ?venue .
-        ?venue <http://purl.org/dc/terms/publisher> ?publisher .
-      }
-      OPTIONAL {
-        ?publication <https://semopenalex.org/property/hasVolume> ?volume .
-      }
-      OPTIONAL {
-        ?publication <https://semopenalex.org/property/hasIssue> ?issue .
-      }
-      OPTIONAL {
-        ?publication <http://purl.org/spar/fabio/hasPublicationYear> ?publicationYear .  
-      }
-      
-    } 
-    GROUP BY ?title ?publication ?publicationYear ?cited ?publisher ?volume ?issue' 
-    number-of-displayed-rows="10" 
-    tuple-template='
-      <div class="publication">
-        <div class="publication__top">
-          <semantic-link iri="{{publication.value}}">
-            <div class="publication__title">{{title.value}}</div>
-          </semantic-link>
-          <div class="publication__author" style="width: 100%;">
-            <mp-text-truncate truncate="..." style="width: inherit;">
-              By {{authors.value}}
-            </mp-text-truncate>
+  <mp-event-target-template-render id="publications" template='{{>tmp}}'>
```

Comment:
I strongly recommend to add some documentation (backend templ comments) to this that describe the variables being passed in (and their intended use), i.e the mode